llvm/lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -233,7 +233,7 @@ static cl::opt<PreferPredicateTy::Option> PreferPredicateOverEpilogue(
                          "prefers tail-folding, don't attempt vectorization if "
                          "tail-folding fails.")));
 
-static cl::opt<TailFoldingStyle> ForceTailFoldingStyle(
+cl::opt<TailFoldingStyle> ForceTailFoldingStyle(
```

Comment:
I don't think this is the right approach to check ForceTailFoldingStyle, better to use `CM.foldTailWithEVL()`

Reply:
Fixed, thanks.